payroll-processor-client/src/app/employee/employee-list/employee-list.component.ts

```diff
@@ -28,4 +29,25 @@ export class EmployeeListComponent implements OnInit, OnDestroy {
   ngOnInit() {}
 
   ngOnDestroy() {}
+
+  updateEmployeeStatus(id: string) {
+    let employee: Employee;
+
+    this.employees
```

Comment:
A much easier way to get data out of a store in this situation is to use `this.query.getValue()`.

Usually for rendering / binding state to a component / template, we want to use Observables because they will keep the UI consistent with changes in state.

However, for user generated events (like typed characters, clicks, hovers, ect...), it's ok to use values directly (an in Akita's case, pull them directly out of the store) because we only care about what the app state is at the moment of the user's interaction - not what the state will be like in the future.